phpunit.xml.dist

```diff
@@ -22,5 +22,9 @@
         <include>
             <directory suffix=".php">src</directory>
         </include>
+        <exclude>
+            <directory>src/DependencyInjection</directory>
+            <directory>src/Resources</directory>
```

Comment:
I agree that the `src/Resource` should be excluded. But the `src/DependencyInjection` contains code that can be (and should be) tested.

```suggestion
            <directory>src/DependencyInjection</directory>
```